docs/HowTo/Find-and-Connect/Managing-Peers.md

```diff
@@ -2,72 +2,91 @@
-is being used to reduce transaction sharing traffic.
+* Determine which protocol level peers are communicating with.
+  For example, you can see if `"version": 65` is being used to reduce transaction sharing traffic.
 
-## Node connections
+## List node connections
 
-The default logging configuration does not list node connection and disconnection messages.
+The default logging configuration doesn't list node connection and disconnection messages.
+To enable listing them, set the [`--logging`](../../Reference/CLI/CLI-Syntax.md#logging) option to `DEBUG`.
+For more verbosity, set the option to `TRACE`.
 
-To enable listing of node connection and disconnection messages, specify the
-[`--logging`](../../Reference/CLI/CLI-Syntax.md#logging) option `--logging=DEBUG`. For more
-verbosity, specify `--logging=TRACE`.
-
-The console logs connection and disconnection events when the log level is `DEBUG` or higher. If
-the message `Successfully accepted connection from ...` displays, connections are getting through
-the firewalls.
+The console logs connection and disconnection events when the log level is `DEBUG` or higher.
+If the message `Successfully accepted connection from ...` displays, connections are getting through the firewalls.
 
 !!! example "Sample log output"
 
-    `2018-10-16 12:37:35.479-04:00 | nioEventLoopGroup-3-1 | INFO | NettyP2PNetwork | Successfully accepted connection from 0xa979fb575495b8d6db44f750317d0f4622bf4c2aa3365d6af7c284339968eef29b69ad0dce72a4d8db5ebb4968de0e3bec910127f134779fbcb0cb6d3331163c`
+    ```bash
+    2018-10-16 12:37:35.479-04:00 | nioEventLoopGroup-3-1 | INFO | NettyP2PNetwork | Successfully accepted connection from 0xa979fb575495b8d6db44f750317d0f4622bf4c2aa3365d6af7c284339968eef29b69ad0dce72a4d8db5ebb4968de0e3bec910127f134779fbcb0cb6d3331163c
+    ```
 
-## Limiting remote connections
+## Disable discovery
 
-In private and permissioned networks with a level of trust between peers, enabling the
-[remote connection limits](../../Reference/CLI/CLI-Syntax.md#remote-connections-limit-enabled)
-is unnecessary and might adversely affect the speed at which nodes can join the network.
-Limiting remote connections may cause a clique of nodes to form when the number of nodes in the network is slightly
-higher than [`--max-peers`](../../Reference/CLI/CLI-Syntax.md#max-peers).
-The nodes in this clique are all connected to each other and cannot accept more connections.
+To disable P2P discovery, set the
+[`--discovery-enabled`](../../Reference/CLI/CLI-Syntax.md#discovery-enabled) option to `false`.
 
-In public networks, enabling the remote connections limit is necessary to prevent eclipse attacks, especially when
-using [`--sync-mode`](../../Reference/CLI/CLI-Syntax.md#sync-mode) and
-[`--fast-sync-min-peers`](../../Reference/CLI/CLI-Syntax.md#fast-sync-min-peers).
+With discovery disabled, peers that have already discovered or are otherwise configured to connect
```

Comment:
"With discovery disabled, peers that have already been discovered..."

Reply:
This one looks complicated. I read it 3 times and still not sure to get it.

Suggestion: 
```
When discovery is disabled, peers can't open connections with the node
unless they were previously discovered or manually peered (for example,
using [`admin_addPeer`](../../Reference/API-Methods.md#admin_addpeer))
```

Not sure that it's the expected meaning though, feel free to ignore this suggestion if I misunderstood the initial sentence.